packages/patternfly-4/react-core/src/components/Page/PageHeader.d.ts

```diff
@@ -7,7 +7,7 @@ export interface PageHeaderProps extends HTMLProps<HTMLDivElement> {
   toolbar?: ReactNode;
   avatar?: ReactNode;
   topNav?: ReactNode;
-  linkComponent?: ReactNode;
+  logoComponent?: ReactNode;
```

Comment:
This one is my fault, sorry!

Reply:
why is prop name changing here?

Reply:
Zack asked me to include this based on: https://github.com/patternfly/patternfly-react/pull/2117